node-bootstrapper/main.go

```diff
@@ -18,23 +20,64 @@ import (
 	"github.com/Azure/agentbaker/pkg/agent/datamodel"
 )
 
+// Some options are intentionally non-configurable to avoid customization by users
+// it will help us to avoid introducing any breaking changes in the future.
 const (
+	ConfigVersion                  = "v0"
 	DefaultAksAadAppID             = "6dae42f8-4368-4678-94ff-3960e28e3630"
-	ReadOnlyWorld      os.FileMode = 0644
+	LogFile                        = "/var/log/azure/node-bootstrapper.log"
```

Comment:
im assuming we're planning on linux-only for the time being?

Reply:
Good point, we will need to come up with something for windows later.

But before that I need to figure out a way to quickly run windows tests, so I can iterate somehow.